tests/filecheck/transforms/stream-snaxify.mlir

```diff
@@ -0,0 +1,22 @@
+// RUN: ./compiler/snax-opt --split-input-file %s -p insert-accfg-op{accelerator=snax_alu},stream-snaxify | filecheck %s
```

Comment:
I'm wondering if it would make more sense to write the accfg-op explicitly here to make this test more standalone?
It does increase duplication a bit though :(

Reply:
Hmm than the filecheck becomes really cluttered, and I don't see a downside to this method.

Reply:
Yeah I agree!